web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml

```diff
@@ -61,17 +61,23 @@
       <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
       <factory-id>addonContainer</factory-id>
     </container>
-    <container id="middle-topNavigation-container" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
-      <name>middle-topNavigation-container</name>
+    <container id="ChatPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
+      <name>ChatPortlet</name>
+      <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
+      <factory-id>addonContainer</factory-id>
+    </container>
+    <container id="NotificationTopBar" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
+      <name>NotificationTopBar</name>
+      <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
       <factory-id>addonContainer</factory-id>
     </container>
     <container id="AppCenterAppLauncherPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
       <name>AppCenterAppLauncherPortlet</name>
       <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
       <factory-id>addonContainer</factory-id>
     </container>
-    <container id="NotificationTopBar" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
-      <name>NotificationTopBar</name>
+    <container id="TimeTrackerPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
```

Comment:
this  "TimeTrackerPortlet" ID should not appear in the product. This container should be a generic extension point for 3rd party addons.

CC @AzmiTouil

Reply:
@SaraBoutej you can use a general container id and name like for middle-topNavigation-container